meteor-app/imports/ui/pages/workspace/component.js

```diff
@@ -214,15 +218,13 @@ export default class WorkspacePage extends React.Component {
                   </div>
                   <div className="layer-opacity-row">
                     <label>Opacity: </label>
-                    <input
+                    <Slider
                       className="input-slider--layer-opacity"
-                      type="range"
-                      min="0"
-                      max="255"
-                      step="1"
+                      min={0}
+                      max={255}
                       value={layer.opacity * 255}
                       data-layer-index={layerIndex}
-                      onChange={this._bound_layerOpacityOnChange}
+                      onChange={newValue => this._bound_layerOpacityOnChange(newValue, layerIndex)}
```

Comment:
Not good practice. Remember the relaying function I mentioned? 

```JavaScript
const relay = (func) => {
  return function (...args) {
    return func(this, ...args);
  };
};
```

This function relays the context object to the first argument.